src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+.form {
+  box-sizing: border-box;
+  position: relative;
+  background: #fff;
+  border-radius: 4px;
+}
+
+[data-qa="small"] {
+  padding: 0;
+  height: 42px;
+}
+
+.small-icon {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+  position: absolute;
+}
+
+[data-qa="big"] {
+  margin: 20px 0;
+  height: 70px;
+}
+
+.icon {
+  left: 26px;
+  top: 25px;
+  position: absolute;
+}
+
+[data-qa="keypress"] {
```

Comment:
![image](https://user-images.githubusercontent.com/97547824/206840984-ab8ecce9-0f9f-43a8-90ac-f280a42f1f82.png)
Fix it everywhere